### DISCUSSION TITLE
OpenStore Order Questions

### DISCUSSION THREAD

**Discussion by alex-pommier** [openstore-ecommerce/OpenStore#295] [Q&A]
Emails:
When a customer successfully purchases a product neither the admin nor the customer is receiving any confirmation or status update email from the system. Does OpenStore send emails? 

Order Status:
Additionally on return from a successful paypal order the status of the order is "Payment Not Verified" where it should be "Payment Ok" but that is not happening automatically an admin must switch it manually for every order which is quite tedious. Is this the normal workflow?

**Comment by DNNMonster** [comment]
Yes OS should send out emails.  It will use the DNN admin settings for the site's SMTP server.  If your DNN site is sending emails then OS should also be able to.    You can view the code here that is doing the work: https://github.com/openstore-ecommerce/OpenStore/blob/4af26cc2c45da5a4c524526f3c7b2379f4be11a7/Components/NBrightBuyUtils.cs#L634

I would also double check that the OS Back Office store admin merchant details settings are accurate for the Email Addresses associated with Administrator and Manager.

You should be able to update an order status which should send out email and update the audit log for a easy test.

For the PayPal default status on a successful payment...  I think there may be some logic to not automatically setting the status to PaymentOK.  You can customize this pretty easy but I am assuming not all payments that return via the PayPal workflow are assured to have cleared.  I'm not 100% sure about this so I'm open to being corrected.  I think the idea is that you still may need to verify some payments otherwise you risk sending out bogus order status emails.
